src/components/forms/SubmitTemplate/server-action.ts

```diff
@@ -0,0 +1,40 @@
+'use server';
+
+import { redirect, RedirectType } from 'next/navigation';
+import { getSession, saveTemplate } from '@utils/form-actions';
+import { createTemplateFromSession, validateTemplate } from '@domain/templates';
+import { logger } from '@utils/logger';
+
+export async function submitTemplate(formData: FormData) {
+  const sessionId = String(formData.get('sessionId'));
```

Comment:
The other server actions get this data a little differently, using `formData.get(...` and `schema.safeParse(...`

e.g.
src/components/forms/CreateNhsAppTemplate/server-action.ts#L52

Reply:
We don't need a return but it helps with unit tests. If we don't return we end up having to mock out the functionality to cause the code to exit early - which is implemented in `__tests__/app/create-nhs-app-template/page.test.tsx`.

I think returning makes the unit tests a bit easier. But happy to follow the established pattern.